app/src/main/res/values/strings.xml

```diff
@@ -1,12 +1,18 @@
 <resources>
     <string name="app_name">FaceRecognition</string>
 
-    <string name="model_Facenet">"Facenet"</string>
-
     <string name="main_NoPermissions_hint">Brakujące kluczowe uprawnienia. Aplikacja zostanie zamknięta.</string>
 
+    <string name="default_model_name">Facenet</string>
+    <string name="user_settings_name"></string>
     <string name="addface_Username_hint">Nazwa użytkownika</string>
     <string name="addface_UsernameNotGiven_toast">Nie wprowadzono nazwy!</string>
     <string name="addface_UserAdded_toast">Dodano użytkownika %s</string>
     <string name="addface_NotOneFaceFound_toast">"Brak lub więcej niż jedna twarz na zdjęciu"</string>
+    <string name="title_activity_settings">SettingsActivity</string>
+
+    <string name="choose_which_models_should_be_used_in_benchmark_mode">Choose which models should be used in benchmark mode</string>
```

Comment:
Fix this names to some standard format (and shorten them to make their usage easier).
Propsed format (also used above) is: (target_location)_(string_name)_(string_type) what gives for example addface_UserAdded_toast.
"choose_model_to_be_used_in_app" should be changed to something like "settings_ChooseUserModel_hint". It contains way more information and is almost the same length.

This comment applies to all the strings added, as they all seem to follow no convention.